<launch>
	
	<arg name="marker_size" default="11.8834" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />
	<arg name="cam_image_topic" default="/webcam/image_raw" />
	<arg name="cam_info_topic" default="/webcam/camera_info" />	
	<arg name="output_frame" default="/web_camera" />


	 <group ns="/webcam">

	<node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />

	<node name="webcam" pkg="uvc_camera" type="uvc_camera_node"> <param name="frame_id" value="web_camera" />  </node>

	</group>

</launch>